packages/kolibri-tools/lib/webpack.config.base.js

```diff
@@ -149,7 +149,7 @@ module.exports = (data, { mode = 'development', hot = false } = {}) => {
           options: {
             objectAssign: 'Object.assign',
           },
-          exclude: /node_modules\/vue/,
+          exclude: /node_modules\/(vue|kolibri-design-system\/lib\/material-svg)/,
```

Comment:
buble will get mad that the precompiled svg files use `with` so we'll exclude them specifically

Reply:
this can be reverted now